vitrina/datasets/templates/vitrina/datasets/detail.html

```diff
@@ -1,7 +1,242 @@
+
+                <p class="dataset-meta-title">{% translate "Organizacija" %}</p>
+                <p class="dataset-meta-value organization-title"><a
+                            href="/organizations/{{ organization.slug }}">{{ organization.title }}</a></p>
+
+                <p class="dataset-meta-title">{% translate "El. paštas" %}</p>
+                <p class="dataset-meta-value"><a href="mailto:{{ organization.email }}">{{ organization.email }}</a></p>
+
+                <p class="dataset-meta-title">{% translate "Telefonas" %}</p>
+                <p class="dataset-meta-value">{{ organization.phone }}</p>
+
+                <p class="dataset-meta-title">{% translate "Tinklalapis" %}</p>
+                <p class="dataset-meta-value"><a href="{{ organization.website }}"
+                                                 target="_blank">{{ organization.website }}</a></p>
+
+                {% if coordinator %}
+                    <p class="dataset-meta-title">{% translate "organization.coordinator" %}</p>
+                    {#<p class="dataset-meta-value"><span>{{ coordinator.getPrettyName() }}</span></p>#}
+                    <p class="dataset-meta-value"><a href="mailto:{{ coordinator.email }}">{{ coordinator.email }}</a>
+                    </p>
+                    {% if coordinator.phone %}
+                        <p class="dataset-meta-value"><span>{{ coordinator.phone }}</span></p>
+                    {% endif %}
+                {% endif %}
+
+                <hr class="custom-line-margin">
+                <p class="dataset-meta-title m-b-md">{% translate "Dalintis socialiniuose tinkluose" %}</p>
+                <p class="social-icons m-t-md">
+                    <a target="_blank"
+                       href="https://www.facebook.com/sharer/sharer.php?u=https://data.gov.lt/dataset/{{ dataset.slug }}"
+                       class="dataset-meta-link">
+                        <span class="icon is-gray">
+                            <i class="fab fa-facebook-square fa-2x"></i>
+                        </span>
+                    </a>
+                    <a target="_blank"
+                       href="https://www.linkedin.com/sharing/share-offsite/?url=https://data.gov.lt/dataset/{{ dataset.slug }}&title={{ dataset.title }}"
+                       class="dataset-meta-link m-l-sm">
+                        <span class="icon is-gray">
+                            <i class="fab fa-linkedin fa-2x"></i>
+                        </span>
+                    </a>
+                    <a target="_blank"
+                       href="http://twitter.com/share?text={{ dataset.title }}&url=https://data.gov.lt/dataset/{{ dataset.slug }}"
+                       class="dataset-meta-link m-l-sm">
+                        <span class="icon is-gray">
+                            <i class="fab fa-twitter-square fa-2x"></i>
+                        </span>
+                    </a>
+                </p>
+                {% if user %}
```

Comment:
```suggestion
                {% if user.is_authenticated %}
```

See: [Athentication data in templates](https://docs.djangoproject.com/en/3.2/topics/auth/default/#authentication-data-in-templates).